src/routes/Routing.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { useEffect } from 'react'
+import { Route, Routes, useNavigate } from 'react-router-dom'
+import ServerBar from '../discord/Navbar/ServerBar'
+import Checking from '../middlewares/Checking'
+import News from '../News/News'
+
+function Redirect({to}){
+  const navigate=useNavigate()
+  useEffect(()=>{
+    navigate(to)
+  },[navigate,to])
+
+  return;
+}
+
+function Routing() {
```

Comment:
Use arrow functions to keep the logic the same throughout the codebase.

Reply:
Now I have changed this.